src/ensembl/src/content/app/browser/zmenu/ZmenuContent.test.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import { mount } from 'enzyme';
+import faker from 'faker';
+
+import ZmenuContent, {
+  ZmenuContentProps,
+  ZmenuContentItem,
+  ZmenuContentItemProps
+} from './ZmenuContent';
+
+import configureStore from 'src/store';
+import { Markup } from './zmenu-types';
+import { createZmenuContent } from 'tests/fixtures/browser';
+
+describe('<ZmenuContent />', () => {
+  afterEach(() => {
+    jest.resetAllMocks();
+  });
+
+  const store = configureStore();
+  const defaultProps: ZmenuContentProps = {
+    content: createZmenuContent()
+  };
+
+  const wrappingComponent = (props: any) => (
+    <Provider store={store}>{props.children}</Provider>
+  );
+
+  const mountZmenuContent = () =>
+    mount(<ZmenuContent {...defaultProps} />, {
+      wrappingComponent
+    });
+
+  describe('rendering', () => {
+    test('renders zmenu content feature', () => {
+      const wrapper = mountZmenuContent();
+      expect(wrapper.find('.zmenuContentFeature')).toHaveLength(1);
+    });
+
+    test('renders zmenu content line', () => {
+      const wrapper = mountZmenuContent();
+      expect(wrapper.find('.zmenuContentLine')).toHaveLength(1);
+    });
+
+    test('renders zmenu content block', () => {
+      const wrapper = mountZmenuContent();
+      expect(wrapper.find('.zmenuContentBlock')).toHaveLength(2);
+    });
```

Comment:
`toHaveLength` aren't very informative tests. Do we know what's actually rendered in those elements?

Reply:
They aren't worth testing either so I'll remove these test cases

Reply:
Wow-wow, these tests are actually useful; they are showing that data passed to ZmenuContent ends up in correct places.
Only instead of checking the number of correct places, the test should check that these places contain correct data.

Reply:
Yes, however removed these test cases since it is useless testing whether they exist or not. I will write test cases to test the individual child components instead.